src/main/java/com/labC/libraryCheckoutSystem/UserServiceImpl.java

```diff
@@ -0,0 +1,36 @@
+package com.labC.libraryCheckoutSystem;
+
+import org.apache.catalina.User;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.Optional;
+
+public class UserServiceImpl implements UserService{
+    @Autowired
+    UserRepo userRepo;
+    @Override
+    public Optional<User> getUserById(Long id) {
+        return userRepo.findById(id);
+    }
+
+    @Override
+    public User getUserByName(String name) {
+        return null;
+    }
+
+    @Override
+    public User getUserByEmail(String email) {
+        return null;
+    }
```

Comment:
Be mindful of these implementations. We will have to decide if we want to have you fully implement these methods or not before pulling into dev.